ksql-engine/src/main/java/io/confluent/ksql/schema/registry/KsqlSchemaRegistryClientFactory.java

```diff
@@ -35,20 +38,24 @@
 
   private final SslFactory sslFactory;
   private final Supplier<RestService> serviceSupplier;
+  private final Map<String, Object> schemaRegistryClientConfigs;
 
   public KsqlSchemaRegistryClientFactory(final KsqlConfig config) {
     this(config, () -> new RestService(config.getString(KsqlConfig.SCHEMA_REGISTRY_URL_PROPERTY)),
-         new SslFactory(Mode.CLIENT));
+         new SslFactory(Mode.CLIENT),
+         config.originalsWithPrefix(KsqlConfig.KSQL_SCHEMA_REGISTRY_PREFIX));
 
     // Force config exception now:
     config.getString(KsqlConfig.SCHEMA_REGISTRY_URL_PROPERTY);
   }
 
   KsqlSchemaRegistryClientFactory(final KsqlConfig config,
                                   final Supplier<RestService> serviceSupplier,
-                                  final SslFactory sslFactory) {
+                                  final SslFactory sslFactory,
+                                  final Map<String, Object> schemaRegistryClientConfigs) {
```

Comment:
`schemaRegistryClientConfigs`  can be dropped. You can get this from `config`, (as is already happening for `sslFactory`):

```
KsqlSchemaRegistryClientFactory(final KsqlConfig config,
                                final Supplier<RestService> serviceSupplier,
                                final SslFactory sslFactory) {
    this.sslFactory = sslFactory;
    this.serviceSupplier = serviceSupplier;
    this.schemaRegistryClientConfigs = ImmutableMap.copyOf(
            config.valuesWithPrefixOverride(KsqlConfig.KSQL_SCHEMA_REGISTRY_PREFIX));
    this.sslFactory.configure(schemaRegistryClientConfigs);
}
```